### ISSUE TITLE
WAKFU does not detect the installed JRE

### ISSUE THREAD

**Issue by Leeo97one** [solus-project/linux-steam-integration#45]
When the game WAKFU is launched from LSI Steam **with the redirect module enabled** (`use-libredirect = true`), it seems to not detect the already installed JRE (OpenJDK 8) on my system.
We can see that in the logs at line 61: https://pastebin.com/yCnDjUGd
```
[SCRIPT ] Using java from :/home/leeo/.local/share/Steam/steamapps/common/Wakfu/jre/bin/java amd64
```

When the game is launched from normal Steam or by disabling the redirect module, it works as expected.
Exemple at line 58-59, 62: https://pastebin.com/aLKckVt9
```
[SCRIPT ] Installed JRE:1.8.0.144(amd64)  in : /usr/bin/java
[SCRIPT ] Using system JRE
[SCRIPT ] Using java from :/usr/bin/java amd64
```

**Comment by ikeydoherty** [comment]
OK thats curious - can you rerun with `LSI_DEBUG=1` in the environment so we can figure out if we accidentally blacklisted some java component? Thanks! (more likely a lib mismatch)

**Comment by ikeydoherty** [comment]
Local run:

![screenshot from 2017-12-21 17-06-14](https://user-images.githubusercontent.com/164489/34266511-4e2e7392-e671-11e7-9aaa-42a2e7278540.png)

**Comment by Leeo97one** [comment]
OK so now the game just fails to launch x)
```
[ERROR  ] ERROR: ld.so: object '/home/leeo/.local/share/Steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
[ERROR  ] [32;1m[lsi:vendor_offender][0m intercepting vendor dlopen() [31;1m/home/leeo/.local/share/Steam/steamapps/common/Wakfu/jre/lib/amd64/libzip.so[0m -> [34;1m/usr/lib64/libzip.so[0m
[32;1m[lsi:vendor_offender][0m transforming vendor soname: [31;1mlibbz2.so.1.0[0m -> [34;1mlibbz2.so.1.0.6[0m
[TRACE  ] Error occurred during initialization of VM
Corrupted ZIP library: /home/leeo/.local/share/Steam/steamapps/common/Wakfu/jre/lib/amd64/libzip.so
[INFO   ] Process exited with code 1
```

Here is the full LSI logs: https://pastebin.com/0LqRKfRm

And I'm running Arch Linux:
```
 leeo  ~  uname -a
Linux leeo-arch-msi 4.14.7-1-ARCH #1 SMP PREEMPT Sun Dec 17 19:13:39 UTC 2017 x86_64 GNU/Linux
```

**Comment by ikeydoherty** [comment]
Aah I see I see. We're blindly allowing it to redirect libzip to the host libzip which has issues.

I know why this wasn't seen before - Arch Linux doesn't split the devel packages so you have the `.so` link within `/usr/lib` - which I don't have here.. I'm gonna need to make it a bit more intelligent and not allow redirection on unversioned links.

**Comment by Leeo97one** [comment]
Oh OK that sounds good ^_^
But do you think it will also fixes the Java detection?

**Comment by ikeydoherty** [comment]
Yeah. As an experiment temporarily move `/usr/lib/libzip.so` out of the way and launch again, it should work. Then move it back when I get the fixes in ^^

**Comment by Leeo97one** [comment]
OK this is great 😀
Thanks for all!
